lib/custom_code_box.dart

```diff
@@ -124,35 +128,88 @@ class InnerField extends StatefulWidget {
 }
 
 class _InnerFieldState extends State<InnerField> {
-  CodeController? _codeController;
+  List<CodeController?> _codeControllers = [];
+  List<int> sumStrBeforeBlock = [];
```

Comment:
Please, change the name of this field (`sumStrBeforeBlock`) into something more readable, or leave a comment

Reply:
Renamed